Podcast/FeedElement.swift

```diff
@@ -10,44 +10,44 @@ import Foundation
 import SwiftyJSON
 
 enum FeedContext: String {
-    case followingRecommendation = "FOLLOWING_RECOMMENDATION"
-    case followingSubscription = "FOLLOWING_SUBSCRIPTION"
-    case newlyReleasedEpisode = "NEW_SUBSCRIBED_EPISODE"
+    case followingRecommendation = "FOLLOWING_RECOMMENDATION" // following recommends a new episode
+    case followingSubscription = "FOLLOWING_SUBSCRIPTION" // following subscribes to new series
+    case newlyReleasedEpisode = "NEW_SUBSCRIBED_EPISODE" // series releases a new episode
 }
 
 class FeedElement: NSObject {
     
     var context: FeedContext // The type of FeedElement this object is
-    var time: Date // The time at which the content of this FeedElement was created
-    var contextSupplier: NSObject // The model that created this content
-    var content: NSObject // The model holding the main information for this FeedElement
+    var time: Date // The time at which the subject of this FeedElement was created
+    var supplier: NSObject // The model that created this subject (ex: supplier = Series when context = .newlyReleasedEpisode)
+    var subject: NSObject // The model holding the main information for this FeedElement (ex: subject = Episode when context = .newlyReleasedEpisode)
     
-    init(context: FeedContext, time: Date, contextSupplier: NSObject, content: NSObject) {
+    init(context: FeedContext, time: Date, supplier: NSObject, subject: NSObject) {
         self.context = context
         self.time = time
-        self.contextSupplier = contextSupplier
-        self.content = content
+        self.supplier = supplier
+        self.subject = subject
         super.init()
     }
     
     convenience init(json: JSON) {
-        var contextSupplier: NSObject
-        var content: NSObject
+        var supplier: NSObject
+        var subject: NSObject
         let context = FeedContext(rawValue: json["context"].stringValue)!
```

Comment:
Should this be forced to type `FeedContext`? In case there are errors from the json

Reply:
Hmm yeah if this fails then we're just gonna crash, we might want the whole init to be optional in that case.

Reply:
Yes, agreed.